_maps/shuttles/emergency_fame.dmm

```diff
@@ -0,0 +1,1908 @@
+"mz" = (
+/obj/effect/spawner/structure/window/reinforced/shuttle,
+/turf/open/floor/plating,
+/area/shuttle/escape)
+"mT" = (
+/obj/structure/sign/painting/large/library,
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/shuttle/escape)
+"mZ" = (
+/obj/item/kirbyplants/organic/plant1,
+/turf/open/floor/mineral/silver,
+/area/shuttle/escape)
+"ni" = (
+/obj/item/scalpel{
+	pixel_y = 12
+	},
+/obj/item/circular_saw,
+/obj/item/retractor{
+	pixel_x = 4
+	},
+/obj/item/hemostat{
+	pixel_x = -4
+	},
+/obj/item/clothing/gloves/latex,
+/obj/item/clothing/mask/surgical,
+/obj/item/surgicaldrill,
+/obj/item/cautery,
+/obj/structure/table/glass,
+/obj/structure/extinguisher_cabinet/directional/south,
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/escape)
+"nx" = (
+/obj/machinery/power/shuttle_engine/huge{
+	dir = 8
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/escape)
+"oc" = (
+/obj/structure/closet/emcloset/anchored,
+/obj/effect/turf_decal/bot,
+/turf/open/floor/iron/dark,
+/area/shuttle/escape)
+"ok" = (
+/obj/structure/window/reinforced/spawner/directional/east,
+/obj/structure/chair{
+	dir = 8
+	},
+/turf/open/floor/mineral/titanium/white,
+/area/shuttle/escape)
+"oz" = (
+/obj/machinery/door/window/brigdoor/security/holding/left/directional/west,
+/obj/effect/decal/cleanable/blood/tracks{
+	dir = 4
+	},
+/turf/open/floor/fakebasalt,
+/area/shuttle/escape/brig)
+"oB" = (
+/obj/effect/spawner/structure/window/reinforced/shuttle,
+/turf/open/floor/mineral/titanium,
+/area/shuttle/escape)
+"pg" = (
+/obj/machinery/suit_storage_unit/standard_unit,
+/obj/machinery/light/small/directional/west,
+/turf/open/floor/plating,
+/area/shuttle/escape)
+"px" = (
+/obj/structure/closet/firecloset,
+/obj/effect/turf_decal/bot,
+/turf/open/floor/iron/dark,
+/area/shuttle/escape)
+"qd" = (
+/obj/effect/turf_decal/siding{
+	dir = 9
+	},
+/turf/open/floor/carpet/royalblue,
+/area/shuttle/escape)
+"qj" = (
+/obj/machinery/computer/station_alert{
+	dir = 8
+	},
+/turf/open/floor/iron/dark,
+/area/shuttle/escape)
+"qn" = (
+/obj/structure/chair/sofa/corp/corner,
+/turf/open/floor/mineral/silver,
+/area/shuttle/escape)
+"qr" = (
+/obj/item/reagent_containers/cup/glass/trophy/bronze_cup,
+/turf/open/floor/carpet/red,
+/area/shuttle/escape)
+"qx" = (
+/obj/structure/curtain/cloth/fancy,
+/turf/open/floor/mineral/titanium,
+/area/shuttle/escape)
+"qG" = (
+/obj/structure/chair/sofa/corp{
+	dir = 1
+	},
+/turf/open/floor/mineral/silver,
+/area/shuttle/escape)
+"qM" = (
+/obj/structure/sign/nanotrasen,
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/shuttle/escape)
+"qW" = (
+/obj/effect/turf_decal/siding{
+	dir = 10
+	},
+/turf/open/floor/carpet/royalblue,
+/area/shuttle/escape)
+"rH" = (
+/obj/structure/chair/sofa/corp/corner{
+	dir = 1;
+	name = "couch";
+	desc = "Looks like someone threw it out. Covered in donut crumbs."
+	},
+/turf/open/floor/mineral/silver,
+/area/shuttle/escape)
+"rV" = (
+/obj/structure/sign/picture_frame/showroom/one,
```

Comment:
Currently these picture frames will mirror the persistence from the station showroom and will possibly end up overwriting each other when persistence is saved

What you may want to do instead is give each of them a unique persistence_id (i.e. fameship1, fameship2 etc.), no need to make them subtypes just varedits should be enough

Reply:
Thanks for pointing this out.  Fixed with https://github.com/tgstation/tgstation/pull/77074/commits/70fb87d672493485b1d32c22000713a17d1df06d